iphone/Maps/Core/TextToSpeech/MWMTextToSpeech.mm

```diff
@@ -286,16 +288,12 @@ + (void)removeObserver:(id<MWMTextToSpeechObserver>)observer {
 
 namespace tts
 {
-std::string translatedTwine(std::string const & twine)
+std::string translateLocale(std::string const & localeString)
 {
-  auto const & list = routing::turns::sound::kLanguageList;
-  auto const it =
-      find_if(list.begin(), list.end(),
-              [&twine](std::pair<std::string, std::string> const & pair) { return pair.first == twine; });
-
-  if (it != list.end())
-    return it->second;
-  else
-    return "";
+  NSString *nsLocaleString = [NSString stringWithUTF8String: localeString.c_str()];
+  NSLocale *locale = [[NSLocale alloc] initWithLocaleIdentifier: nsLocaleString];
+  NSString *localizedName = [locale localizedStringForLocaleIdentifier:nsLocaleString];
```

Comment:
```suggestion
  NSString * localizedName = [locale localizedStringForLocaleIdentifier:nsLocaleString];
```

Reply:
Mmh I never get used to Objective-C syntax ^^ but I stick to the convention.